portfolio/src/main/webapp/script.js

```diff
@@ -47,19 +48,11 @@ function createMap() {
 
   map.controls[google.maps.ControlPosition.TOP_RIGHT].push(
     document.getElementById('floating-panel'));
-
   map.controls[google.maps.ControlPosition.TOP_LEFT].push(
     document.getElementById('search-reports'));
-  var autocomplete = new google.maps.places.Autocomplete(
+  autocomplete = new google.maps.places.Autocomplete(
```

Comment:
autocomplete is a confusing name. addressInput or something is clearer

Reply:
Noted, thanks!